.github/workflows/rc-tests.yml

```
@@ -0,0 +1,43 @@
+name: rc-testing
+
+on:
+  workflow_dispatch:
+  schedule:
+    # Weekly Monday 10AM build
+    - cron: '0 10 * * 1'
```

nitpick, but make it run in Thu, as we hope RCs are produced during the week rather than the weekend :)

```suggestion
    # Weekly Thursday 10AM build
    - cron: '0 10 * * 4'
```